backend/code_review_backend/issues/management/commands/load_in_patch.py

```diff
@@ -36,7 +36,7 @@ def load_hgmo_patch(diff):
     resp = requests.get(url)
     resp.raise_for_status()
 
-    patch = Patch.parse_patch(resp.content, skip_comments=False)
+    patch = Patch.parse_patch(resp.content.decode("utf-8"), skip_comments=False)
```

Comment:
Might be better to ignore possible decoding errors

Reply:
why not `resp.text`?

Reply:
Yep, that works too, it should be equivalent to resp.content.decode("utf-8", "replace").

Reply:
i tested in alpine, that works nicely, thanks @jcristau !